server/src/Application/Dtos/Incoming/DtoControllers/CourseDtoController.cs

```diff
@@ -0,0 +1,18 @@
+using StudyZen.Domain.Entities;
+using StudyZen.Application.Dtos;
+
+namespace StudyZen.DtoControllers
+{
+    public class CourseDtoController
+    {
+        public static CourseDto ToDto(Course course)
```

Comment:
Maybe if we have `BaseEntity`, we could have something like `BaseDto` to meet DRY principle, because every entity is going to have `id` and `CreatedBy`, `UpdatedBy`.

Reply:
not sure if it's necessary since some dtos might not have an id, for example if using a composite key; also we might not need createdby and updatedby properties for some apis. Having a base class is a limiting design: https://stackoverflow.com/questions/49002/prefer-composition-over-inheritance

Reply:
I agree with Arminas